src/frontend/screens/MapScreen/MapLayers/RemoteDetectionAlertsLayer.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import MapboxGL from '@rnmapbox/maps';
+
+import {RemoteDetectionAlert} from '@comapeo/schema';
+import {FeatureCollection} from 'geojson';
+import {useRemoteDetectionAlerts} from '../../../hooks/server/remoteDetectionAlert';
+
+const LABEL_FILTER = [
+  'all',
+  [
+    'in',
+    '$type',
+    'Polygon',
+    'LineString',
+    'Point',
+    'MultiLineString',
+    'MultiPolygon',
+    'MultiPoint',
+  ],
+  ['has', 'alert_type'],
```

Comment:
This has to match the key used the corresponding geojson `properties`

```suggestion
  ['has', 'alertType'],
```

Reply:
https://github.com/digidem/comapeo-mobile/pull/895/commits/313a10eafaa3eddf45bb9151022851dd480ef2f1